libs/zzz/ui/src/Components/ZCard.tsx

```
@@ -0,0 +1,37 @@
+export const ZCard = styled(Card, {
+  shouldForwardProp: (prop) => prop !== 'bgt',
+})<StyledCardProps>(({ theme, bgt }) => {
+  if (!bgt) bgt = 'normal'
+  switch (bgt) {
+    case 'light':
+      bgt = 'contentLight'
+      break
+    case 'dark':
+      bgt = 'contentDark'
+      break
+    case 'normal':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)